test/integration/prometheus/generate_test.go

```diff
@@ -128,7 +146,7 @@ func TestPrometheusGenerate(t *testing.T) {
 			// Run with context to stop on test end.
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
-			out, _, err := prometheus.RunSlothGenerate(ctx, config, test.genCmdArgs)
+			out, _, err := prometheus.RunSlothGenerate(ctx, config, test.genCmdArgs, test.input)
```

Comment:
I don't love how this has to be plumbed through, I'd be happy to put the `input` in the `Config`, or something else if you prefer.

Reply:
One other option would be to have `prometheus.RunSlothGenerate()` return an `exec.Cmd` which we can then modify, or `.Run()` as we like :shrug: